### ISSUE TITLE
Make mobile friendly

### ISSUE THREAD

**Issue by MasterOfTheTiger** [heb12/heb12-android#2]
We need to make the text bigger and work on small displays. I will make some changes upstream that will make it easier for it to be done here.

**Comment by Pufflegamerz** [comment]
The json files can't be read locally, so we will have to make another method. One of them is loading the entire bible into a js file.

**Comment by MasterOfTheTiger** [comment]
Can't we use @amazinigmech2418's method of getting it locally? Plus, it seems to be working currently.

**Comment by MasterOfTheTiger** [comment]
I hate to say it, but I think it would be better to build it from scratch for Android with Java.

**Comment by Pufflegamerz** [comment]
Yeah. I tried using the local method. It works with everything but file:///

**Comment by MasterOfTheTiger** [comment]
Here is some open source code that we can look at: https://gitlab.com/hotlittlewhitedog/BibleMultiTheLight/tree/HEAD

**Comment by Pufflegamerz** [comment]
Will the JSON files be stored locally in the app or on the web?

**Comment by MasterOfTheTiger** [comment]
We need to have the program store the Bible locally. However that may be.

**Comment by MasterOfTheTiger** [comment]
This also could probably be useful: https://github.com/matt-cook/bible

**Comment by Pufflegamerz** [comment]
It could look like this: 
![heb12screen](https://user-images.githubusercontent.com/32369330/39603672-db1f2392-4eee-11e8-904d-1e34dccc3eac.png)

**Comment by MasterOfTheTiger** [comment]
Looks great! Like I said, it should probably not be built with HTML5 stuff though.

**Comment by Pufflegamerz** [comment]
Nah. Android bootstrap thingies look cool

**Comment by MasterOfTheTiger** [comment]
Can you elaborate. What do you mean? Are you saying it is a good or bad idea that we should to it with JS?

**Comment by Pufflegamerz** [comment]
I just making it with android studio is fine.

**Comment by Pufflegamerz** [comment]
I made a prototype https://appetize.io/embed/xcg1vrhxu7070ykgefx4n44c5m?device=nexus5&scale=75&orientation=portrait&osVersion=7.1

**Comment by MasterOfTheTiger** [comment]
Yes. That looks great! That is what I want to do. I don't want to build it with web stuff though, but with Java (or anything else that is best to use for an Android app). Android Studio by default uses Java (unless you tell it to use C++).

**Comment by Pufflegamerz** [comment]
Do you know Java?

**Comment by Pufflegamerz** [comment]
I know some C#.

**Comment by MasterOfTheTiger** [comment]
Well, I think we should use Java for the code. Or C++ if it works well. It needs to be developed for Android from scratch though.

**Comment by pufflegamerz** [comment]
Maybe we can use [Kotlin](https://developer.android.com/kotlin/get-started)

**Comment by MasterOfTheTiger** [comment]
Maybe. Are you going to learn it soon? I want to learn Java or something, but right now I am learning C++.
